packages/guideline-blocks-settings/src/components/RichTextEditor/plugins/ButtonPlugin/components/FloatingButton/useVirtualFloatingButton.ts

```diff
@@ -1,6 +1,6 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { UseVirtualFloatingOptions, flip, offset, useVirtualFloating } from '@udecode/plate';
+import { UseVirtualFloatingOptions, flip, offset, useVirtualFloating } from '@frontify/fondue';
```

Comment:
Can you specify the return type of `useVirtualFloatingButton` which is `UseVirtualFloatingOptions`